### PR TITLE
Feature/file upload

### PR DESCRIPTION
**Pull request by lagartoverde**

### CODE REVIEW

**Comment by lagartoverde** [comment]
I have a linting error on editor-document for the indentation, I understand why the error, but I think that's the correct indentation for that block, what can we do about that rule?

File: support/editor-document.js [hunk 1]
```diff
@@ -79,17 +79,54 @@ async function editorDocumentFromUuid( uuid ){
...
+      attachmentsGrouped[attachment.decision] = [attachment];
+    }
+  }
+  const dom = new jsdom.JSDOM( `<body>${documentContent}</body>` );
+  for(let decisionKey in attachmentsGrouped) {
+    const htmlToAdd = generateAttachmentPart(attachmentsGrouped[decisionKey]);
+    const decisionContainer = dom.window.document.querySelector(`[resource="${decisionKey}"]`);
+    decisionContainer.insertAdjacentHTML('beforeend', htmlToAdd);
+  }
+  return dom.window.document.body.outerHTML;
```
Comment (nvdk) [inline support/editor-document.js:109]: you'll want innerHTML here I assume? outerHTML will include the body tag

**Review by nvdk — CHANGES_REQUESTED** [review]
- a small issue with the html output I think 
-  could you add some tests for the attachments so we know for sure that they are annotated correctly?
- still some linting issues in editor-document.js

**Comment by nvdk** [comment]
@lagartoverde small reminder to use a handlebars template for the attachments

**Comment by lagartoverde** [comment]
Totally missed it, sorry crazy day for me, it will be the first thing I do in Monday

**Review by nvdk — APPROVED** [review]

**Pull request merged**